dev/integration_tests/android_splash_screens/splash_screen_load_rotate/README.md

```diff
@@ -0,0 +1,7 @@
+# Splash Demo: Never ending splash animation
```

Comment:
Do we need 2 projects for this? Could it be one APK with 2 activities and then launching it via adb am differently? (not sure how to do it)

Reply:
I think we could technically combine all 3 projects into 1 project. The question would be, should a developer have to move around the "MAIN" category in the manifest by hand before being able to run one of the three projects? If you think that's OK, then I can combine them. If it would be better to keep it easier to run for verification, then we would want to keep them separate. What do you think?

Reply:
Spoke offline. We'll keep the projects separate for now.